siriuspy/siriuspy/orbintlk/main.py

```diff
@@ -1249,8 +1249,8 @@ def _check_configs(self):
         for i, dev in enumerate(self._llrfs):
             if dev.connected:
                 value = _updt_bit(value, 2*i, 0)
-                fim_orbit = dev.fast_interlock_monitor_orbit
-                fim_manual = dev.fast_interlock_monitor_manual
+                fim_orbit = dev['FIMLLRF1-Sts']
```

Comment:
just out of curiosity, why are you abandoning the use of the python property here @anacso17 ?

Reply:
Hi @fernandohds564 , this was a first step for testing, in a next commit I would replace the hard-coded strings by attributes depending on the LLRF, this would make it possible to have different inputs for each LLRF (as we needed the last months)

Reply:
thanks for the explanation!